suricata.yaml.in

```diff
@@ -1560,6 +1560,15 @@ threading:
 luajit:
   states: 128
 
+# YARA options
+yara:
+  # The maximum duration of a scan in seconds. A value of 0
+  # means no timeout. The default is a 1 second timeout.
+  timeout: 1
```

Comment:
Is this a limitation of Yara? What I'm wondering is if ms resolution makes sense here, like 100ms.